webapp/components/PageFooter/PageFooter.spec.js

```diff
@@ -1,10 +1,14 @@
-import { config, mount } from '@vue/test-utils'
+import { mount } from '@vue/test-utils'
 import PageFooter from './PageFooter.vue'
 import linksDefault from '~/constants/links.js'
 
 const localVue = global.localVue
 
-config.stubs['nuxt-link'] = '<span class="nuxt-link"><slot /></span>'
+const stubs = {
+  'nuxt-link': {
+    template: '<span class="nuxt-link"><slot /></span>',
```

Comment:
same

Reply:
Let us say we have something like:
```
<nuxt-link><p>Text I want to check</p></nuxt-link>
```
then I cannot check the `p`-element